html/about.html

```diff
@@ -0,0 +1,233 @@
+        <div class="main-intro-contact-email">
+          <p>
+            Please contact us per Email for any further questions about PLC
+            Global Summit 2022!
+            <span><a href="#">plcsummit2022@us.org</a></span>
+          </p>
+        </div>
+      </section>
+
+      <article class="logo-section">
+        <!-- we look at the logo -->
+        <h2>PLC Global Summit 2022 logo</h2>
+        <p class="logo-description">
+          The logo of PLC Global Summit 2022 was decided through the logo
+          competition from 6 May to 8 June.<span class="display-on-desk">
+            After online public voting and selection committee's judging process
+            and the logo from Kundos Igdaliah won the price. After consultation
+            with the board we optimized the price winning logo for this year's
+            summit as following
+          </span>
+        </p>
+
+        <div class="logo-container">
+          <img src="../assets/summit-logo.png" alt="summit logo" />
+        </div>
+      </article>
+
+      <!-- past global summits  -->
+      <article class="past-summits">
+        <h3 class="past-summits-title">See the past PLC Global Summits</h3>
+        <p class="past-summits-description">
+          Take a look at the last two PLC which took place in Thailand and India
+        </p>
+        <div class="past-summit-container">
+          <div class="thailand visited">
+            <div class="wrapper">
+              <p>2019</p>
+              <p>PLC Global Summit 2019 in Bangkok, Thailand</p>
+            </div>
+          </div>
+          <div class="india visited">
+            <div class="wrapper">
+              <p>2017</p>
+              <p>PLC Global Summit 2017 in Mumbai, India</p>
+            </div>
+          </div>
+        </div>
+      </article>
+    </main>
+
+    <!-- footer -->
+    <footer class="footer">
+      <div class="footer-partner">
+        <h4 class="footer-partner-title">Partners</h4>
+        <div class="footer-partner-container">
+          <img
+            src="../assets/mozilla-1.svg"
+            alt="mozilla"
+            class="footer-partner-image reduce-80"
+          />
+          <img
+            src="../assets/google.svg"
+            alt="Google"
+            class="footer-partner-image google"
+          />
+          <img
+            src="../assets/naver-2.svg"
+            alt="Naver"
+            class="footer-partner-image reduce-80"
+          />
+          <img
+            src="../assets/airbnb.svg"
+            alt="airbnb"
+            class="footer-partner-image"
+          />
+          <img
+            src="../assets/tesla.svg"
+            alt="Tesla"
+            class="footer-partner-image"
+          />
+        </div>
+      </div>
+
+      <div class="footer-copyright">
+        <p class="copy-rights">&copy; PLC- United states</p>
+        <p class="reserved">2022 PLC. <span>Some Rights reserved</span></p>
+      </div>
+    </footer>
```

Comment:
- [ ] The content of the `footer-copyright` div appears skewed to the left and not well balanced. I think it would be better to have it centered so it doesn't look odd. What do you think? 🤔 
- [ ] Please replace the copyright with your logo as shown in the design guideline 🙏